core/core-dotspacemacs.el

```diff
@@ -80,6 +80,9 @@ banner, `random' chooses a random text banner in `core/banners'
 directory. A string value must be a path to a .PNG file.
 If the value is nil then no banner is displayed.")
 
+(defvar dotspacemacs-version-check-enable t
```

Comment:
Yeah for the `-enabled` version I agree, sounds more like a getter than a setter. But I would still prefer the `enable-` version instead of the `-enable`. For instance in `dotspacemacs` there is already [`dotspacemacs-enable-paste-micro-state`](https://github.com/syl20bnr/spacemacs/blob/develop/core/templates/.spacemacs.template#L152), and other imperative forms like `dotspacemacs-active-transparency`, `dotspacemacs-display-default-layout`, …

Anyway, it's not me that will decide on and merge this, so it's just an opinion, propose the version that you want :relaxed: